### DISCUSSION TITLE
Сложността е O(N ln(N))

### DISCUSSION THREAD

**Discussion by ch-hristov** [dimitarminchev/ITCareer#9] [General]
https://github.com/dimitarminchev/ITSliven/blob/8cb1d3333f50b36487e26ffec14186c09aa84015/4.%20ASD/30.06.2018/Problem9/Program.cs#L21

Супер курс. Намерих един малък проблем тук(скоростта е доминирана от сортирането т.е O(N ln N)

(може и по-дизайн да е така, ако не се взима предвид сортирането, но все пак е добре да се каже)

**Comment by dimitarminchev** [comment]
Благодаря, оценката за сложност на алгоритъма е актуализирана.
